source/_components/sensor.dwdwarnapp.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "DWD WarnApp"
```

Comment:
Because it uses the same data as the Android / IOS App "DWD WarnWetter". I could rename it to "DWD Warn Weather" if this would fit better?